pkg/manager/member/pd_member_manager.go

```diff
@@ -905,6 +907,7 @@ func (m *pdMemberManager) collectUnjoinedMembers(tc *v1alpha1.TidbCluster, set *
 			if err != nil {
 				return err
 			}
+			// FIXME: this will show empty PVC UID in status, fix according to PDFailureMember
```

Comment:
Why for `this will show empty PVC UID in status`?

Reply:
this is not accurate, will rephrase